@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fmc/fn_fmcAltitudeHoldEnable.sqf

```diff
@@ -5,7 +5,10 @@ if (_heli getVariable "fza_ah64_altHoldActive" == false) then {
     private _curVel    = vectorMagnitude [velocityModelSpace _heli # 0, velocityModelSpace _heli # 1];
     private _curVelZ   = velocity _heli # 2; 
     private _curAltAGL = ASLToAGL getPosASL _heli # 2;
-    private _curAltASL = getPosASL _heli # 2;
+    if (_curAltAGL > 50) then {
```

Comment:
Should this here be formatted this way? It isn't for display

Reply:
It's there on purpose to ensure that when the Radar Altitude is > 50 ft, instead of trying to maintain a 1 foot altitude resolution, it maintains a 10 foot resolution, which smooths out output to the rotor.